docs/pages/blog/callback-support-in-style-overrides.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Introducing callback support in style overrides
+description: We're excited to introduce callback support for global theme overrides in this minor version update!
+date: 2022-01-20T00:00:00.000Z
```

Comment:
Are we good with the date? Do we want to use the release date of v5.3.0 or the release date?

Reply:
Yep, I will mark the next Monday as the blog post released date.